src/components/ui/sign-description.js

```diff
@@ -0,0 +1,65 @@
+const SignDescription = () => (
+  <div>
+    <div className="imageWrapper">
+      <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/5/005/084/138/2e66790.jpg" />
+    </div>
+    <span>We managed to build, launch and succeed with our Pokemon Go map app in just 48 hours of programming.
+    <author>Artur Czemiel<span>developer & founder, Aexol</span></author>
+    </span>
+
+    <style jsx>{`
+      .imageWrapper {
+        width:48px;
+        height:48px;
+        border-radius:24px;
+        overflow:hidden;
+        margin-bottom:24px;
+        margin-top:20px;
+      }
+
+      .imageWrapper img{
```

Comment:
Learn BEM syntax: https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ and namespaces https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/